lib/mcb.rb

```diff
@@ -112,7 +112,7 @@ def self.generate_apiv2_token(email:, encoding:, secret: nil)
       raise "Secret not provided"
     end
 
-    JWT.encode(payload, secret, encoding)
+    JWT::DecodeService.encode(payload: payload, secret: secret, algorithm: encoding)
```

Comment:
This naming can't be correct. Why isn't there either an `EncodeService.call` and `DecodeService.call` or else something like `TokenService.encode` and `TokenService.decode`. The `call` pattern being what we're settling on.

Reply:
Moved `EncodeService` to `MCB` side